tests/system/feature_store/test_spark_engine.py

```diff
@@ -858,13 +858,15 @@ def test_get_offline_features_with_filter_and_indexes(self):
         assert resp_df[["bad", "department"]].equals(expected_df)
 
     # ML-2802
-    def test_get_offline_features_with_spark_engine(self):
+    @pytest.mark.parametrize("passthrough", [True, False])
+    def test_get_offline_features_with_spark_engine(self, passthrough):
         key = "patient_id"
         measurements = fstore.FeatureSet(
             "measurements",
             entities=[fstore.Entity(key)],
             timestamp_key="timestamp",
             engine="spark",
+            passthrough=passthrough,
```

Comment:
This would pass even if passthrough does nothing though. Can we verify that it's actually doing what it's supposed to?